.idea/workspace.xml

```diff
@@ -0,0 +1,860 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You'll want to skip this file and probably add .idea to the gitignore

Reply:
Done.